src/Makefile.am

```diff
@@ -154,4 +154,10 @@ EXTRA_DIST += \
 	$(eos_updater_resource_deps) \
 	$(NULL)
 
+# Ensure that eos-updater-ctl compiles and runs at least as far as --help
+CLEANFILES += eos-updater-ctl.help-stamp
+all-local: eos-updater-ctl.help-stamp
+eos-updater-ctl.help-stamp: eos-updater-ctl
+	$(AM_V_GEN) $< --help >/dev/null && touch $@
```

Comment:
I was only set on using `eos-updater-ctl --help` as a test case if it was an installed-test, since that would test it in-place on a system, and hence catch issues with dependencies not being installed (or whatever). Calling `--help` as a compile-time step seems quite odd.

In descending order of preference, I like the idea of:
 * A Python test harness which is run by `make check` and is an installed-test, which (for the moment) smoketests by calling `eos-updater-ctl --help` as a subprocess — but in future could be extended to test `eos-updater-ctl`’s behaviour in different ways.
 * A simpler `make check` test akin to this one, which runs `python3-flake8` on `eos-updater-ctl` at compile time. Since you’ve helpfully fixed the `python3-flake8` warnings, adding it as a dependency and using it on future commits won’t add much of a maintenance burden.
 * This.

I should have communicated this preference list earlier, sorry.

Reply:
I thought I had implemented option 2 but in fact it is running flake8 during `make all`. I'll do another round to move it to the `check` stage.